client/verta/.pylintrc

```diff
@@ -351,13 +351,6 @@ max-line-length=100
 # Maximum number of lines in a module
 max-module-lines=1000
 
-# List of optional constructs for which whitespace checking is disabled. `dict-
-# separator` is used to allow tabulation in dicts, etc.: {1  : 1,\n222: 2}.
-# `trailing-comma` allows a space between comma and closing bracket: (a, ).
-# `empty-line` allows space-only lines.
-no-space-check=trailing-comma,
-               dict-separator
-
```

Comment:
This [was removed in `pylint==2.6`](https://pylint.pycqa.org/en/latest/whatsnew/2/2.6/summary.html?#other-changes), and `pylint==2.14` [starts raising an exception for unrecognized options](https://pylint.pycqa.org/en/latest/whatsnew/2/2.14/full.html#what-s-new-in-pylint-2-14-0).